.github/workflows/labeler.yml

```diff
@@ -5,15 +5,17 @@
 name: "Pull Request Labeler"
 
 on:
-- pull_request_target
-- issue_comment
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+  issue_comment:
+    types: [created]
 
 permissions:
   contents: read
 
 jobs:
   triage:
-    if: github.event.repository.name != 'systemd-security'
+    if: github.event.repository.name == 'systemd'
```

Comment:
if you make this `github.repository == 'systemd/systemd'` then it will also avoid running on forks